part_2_dynamic.md

```diff
@@ -128,67 +155,80 @@ end
 
 {% highlight erb %}
 <ul>
-  <% @hobbies.each do |hobby| %>
+  <% @languages.each do |language| %>
     <li>
-      <%= hobby %>
+      <%= language %>
     </li>
   <% end %>
 </ul>
 {% endhighlight %}
 
-Don't forget to do the same for languages.
 
-## If you want to go wild ask your coach about the following code:
+### And now from the database
 
-Option 1: Just bring it back, using a each_with_index
+We're ready for the next level of magic: Let's use the database to store the languages, because editing your code every time you learn a new language is for amateurs.
 
-{% highlight erb %}
-<ul class="list-group">
-  <% @hobbies.each_with_index do |hobby, i| %>
-    <li class="list-group-item <%= i < 3 ? '' : 'hide' %>">
-      <%= hobby %>
-    </li>
-  <% end %>
-</ul>
+{% highlight bash %}
+rails generate scaffold language name:string
+rake db:migrate
+{% endhighlight %}
+
+Check out [http://localhost:3000/languages](http://localhost:3000/languages)
+
+Try out the forms and create, edit and delete a couple of entries.
+
+Now see how we can use the languages stored in the database. For that we tell the controller to load the information from the database. Open the file `app/controllers/pages_controller.rb` in your editor and add the line about languages.
+
+{% highlight ruby %}
+def about
+  @year = Time.now.year
+  @languages = Language.all
+end
 {% endhighlight %}
 
-Option 2: Just bring it back, using `first` and `drop`
+Now go check what the page looks like.
+
+Does it look weird?!
```

Comment:
Not sure I’d put this in the guide. It’s more the thing the coach would say, not the guide

Reply:
If we would go into detail and explain what it does, I think this would be valid step in the guide, but I think it's a detail that doesn't deliver a lot of value in understanding how rails works and has confusion potential. I'll take it out.